test/selenium/sessions_ui_test.js

```diff
@@ -129,6 +132,21 @@ describe('sessions ui', function() {
         });
       });
 
+      it('should fail if password is not set', function() {
```

Comment:
Well, both tests are the same, tapping on de submit/signin without introducing the password, but can change the name to: should fail if password is not typed, for the second test